content/changelog/2024-04-15-new-logs-interface-public-beta.md

```diff
@@ -0,0 +1,23 @@
+---
+title: New logs interface is available in public beta
+date: 2024-04-15
+tags:
+  - clever-tools
+  - cli
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+  - name: Pierre De Soyres
+    link: https://github.com/pdesoyres-cc
+    image: https://github.com/pdesoyres-cc.png?size=40
+description: Lots of new features included!
+excludeSearch: true
+---
+
+Since last year we use a new Logs stack based on Pulsar and Vector, available for applications only, through CLI. Thanks to it, we provide a more reliable service, with new filtering features. It's now available from [the Console](https://console.clever-cloud.com) too! As it's in public beta stage, you'll need to activate it through the "Try the new Logs experience" link. You'll be able to revert this choice if needed. 
```

Comment:
` through CLI` what does it mean? 

```suggestion
Since 2023, applications use a new Logs stack based on Pulsar and Vector, through CLI. Thanks to it, we provide a more reliable service, with new filtering features. It's now available from [the Console](https://console.clever-cloud.com) too! As it's in public beta stage, you'll need to activate it through the "Try the new Logs experience" link. You can revert this choice if needed. 

This new feature is available for **applications only**.
```